crud/stats/local_registry.lua

```diff
@@ -0,0 +1,124 @@
+local dev_checks = require('crud.common.dev_checks')
+local registry_common = require('crud.stats.registry_common')
+
+local registry = {}
+local internal_registry = {}
+
+--- Initialize local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, init is not guaranteed to be idempotent.
+--
+-- @function init
+--
+-- @treturn boolean Returns true.
+--
+function registry.init()
+    internal_registry.spaces = {}
+    internal_registry.space_not_found = 0
+
+    return true
+end
+
+--- Destroy local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, destroy is not guaranteed to be idempotent.
+--
+-- @function destroy
+--
+-- @treturn boolean Returns true.
+--
+function registry.destroy()
+    internal_registry = {}
+
+    return true
+end
+
+--- Get copy of local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, get is not guaranteed to work without init.
+--
+-- @function get
+--
+-- @tparam string space_name
+--  (Optional) If specified, returns table with statistics
+--  of operations on table, separated by operation type and
+--  execution status. If there wasn't any requests for table,
+--  returns {}. In not specified, returns table with statistics
+--  about all existing spaces and count of calls to spaces
+--  that wasn't found.
+--
+-- @treturn table Returns copy of metrics registry (or registry section).
+--
+function registry.get(space_name)
+    dev_checks('?string')
```

Comment:
For stats, if space was specified with `id` instead of `name` (AFAIK, only `crud.len` supports this) I resolve the name from schema and store it by name. Do we really need to also support id resolving here?

Reply:
See registry.get comment